src/mz/Readme.md

```diff
@@ -0,0 +1,127 @@
+# Materialize CLI [WIP]
+
+Rust implementation of critical features for usage and deployment of Materialize platform instances.
+It provides the following actions:
+
+* Login
+* Enable/Delete Regions
+* Shell
+
+## Documentation
+
+### Install
+
+Clone the repository. Once inside, run:
+
+```bash
+cargo build --package mz --release
+```
+
+After a succesfull build. Make the binary executable:
+
+```bash
+chmod +X ./target/release/mz
+
+cd ./target/release
+```
+
+[Install dependencies](#Dependencies)
+
+### Help
+
+Use command help to understand usage:
+
+```bash
+mz help
+```
+
+### Login
+
+The CLI lets you login into the platform and automatically create all it needs to work.
+
+Using the browser:
+
+```bash
+mz login
+```
+
+Using email and password:
+
+```bash
+mz login interactive
+```
+
+### Profiles
+
+The CLI uses `USER_ID`, `SECRET`, and `Email` to interact with the platform. These values are automatically populated through the login command. If you want to retrieve or manually create the profiles file, the file is in the OS config folder:
+
+Linux: `.config/mz/profiles.toml` (TBC)
+macOS (Monterrey): `/Users/{username}/Library/Application Support/mz/profiles.toml`
```

Comment:
minor but I feel like `dirs` actually has the wrong idea here for macOS. IMO it's way less confusing just to use `~/.config` on both Linux and macOS.